src/Data/ByteString/Base64/Internal.hs

```diff
@@ -46,6 +46,14 @@ module Data.ByteString.Base64.Internal
 
 import Data.Bits
 import qualified Data.ByteString as BS
+import Data.ByteString.Base64.Internal.Tail
+import Data.ByteString.Base64.Internal.Utils
+#if WORD_SIZE_IN_BITS == 32
```

Comment:
Don't worry about this CPP for now - i'll defer to a wordsize-agnostic encoding in the #else clause later.